prometheus_rules/README.md

```diff
@@ -1,4 +0,0 @@
-# Prometheus alerts and recording rules
```

Comment:
because it say the same as the name of the directory.

Futhermore, I didn't remove this line but whole REAMDE. The organization if alerts and recording rules chaged by this PR.
From my point of view is worthless to have README here.

Reply:
Ok, let's rather focus on the comments within the files themselves opposed to a separate README file.